datajunction-server/datajunction_server/models/node.py

```diff
@@ -764,15 +764,15 @@ def check_metric(self):
             )
 
         # must query from a single table
```

Comment:
@samredai and @agorajek - we should probably discuss this change. I don't remember the original rationale for keeping metrics limited to querying a single node, because there are times that upstream transforms don't cleanly solve for this case. See the description in the PR for more details. Alternatively we could implement derived metrics to support something like this, but I think it's easier to just relax this restriction until we get around to that.

Reply:
@agorajek So technically it is possible to create a single transform, but I wanted to be able to label the primary key on the transform, which would then be different for `default.regional_level_agg` and `default.national_level_agg`. That's what I meant by "unclean".